sdk/tests/tutorials/properties/test_derived_properties.py

```diff
@@ -0,0 +1,145 @@
+import unittest
+import logging
+import json
+
+import lusid
+
+import lusid.models as models
+from utilities import InstrumentLoader
+from utilities import TestDataUtilities
+
+
+class DerivedPropertyTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # create a configured API client
+        api_client = TestDataUtilities.api_client()
+        cls.scopes_api = lusid.ScopesApi(api_client)
+        cls.property_definitions_api = lusid.PropertyDefinitionsApi(api_client)
+        cls.instruments_api = lusid.InstrumentsApi(api_client)
+        # load instruments from InstrumentLoader
+        instrument_loader = InstrumentLoader(cls.instruments_api)
+        cls.instrument_ids = instrument_loader.load_instruments()
+        # setup logging configuration
+        logging.basicConfig(level=logging.INFO)
+
+    def create_ratings_property(self, *ratings):
+
+        ratings = [*ratings]
+
+        for rating in ratings:
+            property_definition = models.CreatePropertyDefinitionRequest(
+                domain="Instrument",
+                scope="Test-Demo",
```

Comment:
I'm a bit of a stickler for names, can we change the scope to something descriptive like `derived` or `derivedproperties`